MaterialLib/Fluid/ConstantFluidProperty.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   ConstantFluidProperty.h
+ *
+ * Created on August 15, 2016, 12:11 PM
+ */
+
+#ifndef CONSTANTFLUIDPROPERTY_H
+#define CONSTANTFLUIDPROPERTY_H
+
+#include "FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// Constant fluid density properties
+class ConstantFluidProperty : public FluidProperty
+{
+public:
+    ConstantFluidProperty(const double value) : FluidProperty(), _value(value)
+    {
+    }
+
+    /// Get model name.
+    virtual std::string getName() const final { return "constant"; }
+    /// Get property value.
+    /// \param var_vals Variable values in an array. The order of its elements
+    ///                 is given in enum class PropertyVariable.
+    virtual double getValue(const double /* var_vals*/[]) const final
```

Comment:
Doxygen cannot refer to commented variable name. You'll have to add a `(void)var_vals` in the body.